central/cve/node/datastore/datastore_impl_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build sql_integration
```

Comment:
Why does this need this tag?

Reply:
It is a postgres ds test

Reply:
it doesn't use a real postgres instance so doesn't need it

Reply:
Ah, it is using mock stores